sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -29,6 +29,15 @@ export function getGenderCode(gender: string): string {
   return gender === "female" ? "F" : "O";
 }
 
+/**
+ * Returns phone number removing the hyphens.
+ * @param phone phone number to be formatted
+ * @returns phone number removing the hyphens in between. (Eg: 123-456-7890 => 1234567890)
+ */
+export function getFormattedPhone(phone: string): string {
```

Comment:
👍 Just another way of doing it would be 

export function getFormattedPhone(phone: string): string {
  return phone.split("-").join("");
}